tests/components/lastfm/test_sensor.py

```diff
@@ -71,22 +31,40 @@ async def test_update_not_playing(hass: HomeAssistant, lastfm_network) -> None:
     assert state.state == STATE_NOT_SCROBBLING
 
 
-async def test_update_playing(hass: HomeAssistant, lastfm_network) -> None:
+async def test_update_playing(hass: HomeAssistant) -> None:
     """Test update when song playing."""
+    with patch_interface("test", MOCK_TRACK):
+        assert await async_setup_component(hass, sensor.DOMAIN, CONFIG) is True
+        await hass.async_block_till_done()
 
-    lastfm_network.return_value.get_user.return_value = MockUser(
-        Track("artist", "title", MockNetwork("test"))
-    )
+    entity_id = "sensor.test"
 
-    assert await async_setup_component(
-        hass,
-        sensor.DOMAIN,
-        {"sensor": {"platform": "lastfm", "api_key": "secret-key", "users": ["test"]}},
-    )
-    await hass.async_block_till_done()
+    state = hass.states.get(entity_id)
 
-    entity_id = "sensor.test"
+    assert state.state == "Goldband - Noodgeval"
+
+
+async def test_failed_update(hass: HomeAssistant) -> None:
```

Comment:
Use the fixture here instead of the `patch_interface`
```suggestion
async def test_failed_update(
    hass: HomeAssistant, lastfm_network: MockLastFMNetwork
) -> None:
```